.bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "integrity": "",
+  "strip_prefix": "{REPO}-{VERSION}",
+  "url": "https://github.com/{OWNER}/{REPO}/archive/refs/tags/{TAG}.tar.gz"
```

Comment:
Just noticed we have release archive for rules_cc under https://github.com/bazelbuild/rules_cc/releases/download/0.0.2/rules_cc-0.0.2.tar.gz , do you know what's the difference? Which one should we use?

Reply:
From the release notes, it seems we should use this one: https://github.com/bazelbuild/rules_cc/releases